package.json

```diff
@@ -55,7 +61,6 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.3.0",
-    "@types/chai": "^4.3.15",
```

Comment:
presumably this was here for a reason before?

maybe to improve dx? since you'd get definitions in your editor even though the sources are JS (its a dev dependency)

Reply:
chai is already typescript, the types are shipped with it. I commented as much in the relevant commit message

![image](https://github.com/chaijs/chai-http/assets/7128721/6f0e3d3b-9d15-44d7-afd1-ceb3b26b5aef)

Reply:
are you sure? i maintain chai and i still haven't merged my PR which adds typescript definitions 😅

Reply:
well, i clearly misread the types badge on npm. I'll go re-add this

Reply:
aha no worries! gives me motivation to actually finish off sorting the shipped types 🙏